docs/deploying/spacetimedb-standalone.md

```diff
@@ -82,25 +82,64 @@ server {
     listen 80;
     server_name example.com;
 
-    location / {
+    #########################################
+    # By default SpacetimeDB is completely open so that anyone can publish to it. If you want to block
+    # users from creating new databases you should keep this section commented out. Otherwise, if you
+    # want to open it up (probably for dev environments) then you can uncomment this section.
```

Comment:
I believe they would need to uncomment this section _and_ comment out the `location /` block below